### PR TITLE
updates to java-websockets for screencast

### PR DESCRIPTION
**Pull request by indiepopart**
Updates to the post https://developer.okta.com/blog/2019/10/09/java-spring-websocket-tutorial
This updates are for recording the screencast.

### CODE REVIEW

**Comment by mraible** [comment]
Thanks Jimena!

Can you please create a changelog in this post to point to this PR to let people know the post has been updated. You can see an example at the bottom of this post:

https://developer.okta.com/blog/2018/07/19/simple-crud-react-and-spring-boot

You can probably copy/paste the Markdown from this post and use it in yours.

**Comment by indiepopart** [comment]
@mraible Shall I create a pull request for the code repository at https://github.com/oktadeveloper/okta-java-websockets-example?

**Comment by mraible** [comment]
Yes, please do.

**Comment by indiepopart** [comment]
Added change log. Not sure about the date. Created pull request https://github.com/oktadeveloper/okta-java-websockets-example/pull/2

File: _source/_posts/2019-10-09-java-spring-websocket-tutorial.md [hunk 1]
```diff
@@ -548,3 +552,7 @@ To continue learning about WebSocket-related technologies and Spring Framework's
 * [10 Excellent Ways to Secure Your Spring Boot Application](/blog/2018/07/30/10-ways-to-secure-spring-boot)
 
 For more informative tutorials, please [follow @oktadev on Twitter](https://twitter.com/oktadev) and [subscribe to our YouTube channel](https://youtube.com/c/oktadev).
+
+**Changelog:**
+
+* Dic 17, 2020: Update spring-boot version in curl line, to 2.4.0. Update Okta javascript SDK static link in index.html, to version to 4.0.0. Update Tone.js library link. Update the javascript authentication code in auth.js, for new Okta SDK version.
```
Comment (mraible) [inline _source/_posts/2019-10-09-java-spring-websocket-tutorial.md:558]:
```suggestion
* Dec 23, 2020: Updated Spring Boot to version 2.4.0. Updated Okta Auth JS to version 4.0.0. Update Tone.js library link. See the code changes in the [example app on GitHub](https://github.com/oktadeveloper/okta-java-websockets-example/pull/2). Changes to this post can be viewed in [oktadeveloper/okta-blog#495](https://github.com/oktadeveloper/okta-blog/pull/495).
```

File: _source/_posts/2019-10-09-java-spring-websocket-tutorial.md [hunk 2]
```diff
@@ -548,3 +552,7 @@ To continue learning about WebSocket-related technologies and Spring Framework's
 * [10 Excellent Ways to Secure Your Spring Boot Application](/blog/2018/07/30/10-ways-to-secure-spring-boot)
 
 For more informative tutorials, please [follow @oktadev on Twitter](https://twitter.com/oktadev) and [subscribe to our YouTube channel](https://youtube.com/c/oktadev).
+
+**Changelog:**
```
Comment (mraible) [inline _source/_posts/2019-10-09-java-spring-websocket-tutorial.md:556]:
```suggestion
<a name="changelog"></a>
**Changelog:**
```

**Review by mraible — CHANGES_REQUESTED** [review]
Thank you @indiepopart! Please add links in the changelog as suggested.

**Comment by indiepopart** [comment]
@mraible Links added!

**Review by mraible — APPROVED** [review]

**Pull request merged**